public_dropin_environments/python3_keras/fit.sh

```diff
@@ -17,6 +18,10 @@ if [ -f "${weights}" ]; then
     CMD="${CMD} --row-weights-csv ${weights}"
 fi
 
+if [ -f "${sparse_colnames}" ]; then
+    CMD="${CMD} --sparse-column-file ${sparse_colnames}"
```

Comment:
I suggest to create env var for `--sparse-column-file`, once this env var is set, DRUM will automatically pick it up.
Boilerplate is there:
* add into proper record into ArgumentOptionsEnvVars and ArgumentOptionsEnvVars.VALUE_VARS....

What actually happens: in the beginning, if env var is set, parameter will be added into sys.argv.

Reply:
So you won't need this condition in every environment.

Reply:
@jlpolit did it for SKIP_PREDICT

Reply:
I'd like for people to be able to use this on the cli though.

Reply:
it doesn't cancel providing argument explicitly as `--sparse-column-file`